src/main.ts

```diff
@@ -24,17 +25,22 @@ async function run(): Promise<void> {
 
     if (!issueKey) {
       console.log(`JIRA key was not found`);
-      process.exit(0);
+      if (FAIL_PR_WHEN_JIRA_ISSUE_NOT_FOUND === 'true') {
+        console.log('Setting PR status as failed');
```

Comment:
in fact we are just failing the action – settings to fail PR or not are configurable per repo

lets use phrasing "Failing action" or something like that

Reply:
Moved all these statements to the catch block with a simple message. Please take a look